scenarios/scenario-11/README.md

```diff
@@ -102,7 +102,7 @@ To try out the scenario navigate to ```k8s-api-operator-<version>``` directory.
 - By deploying the analytics configmaps, you can enable analytics as follows.
 
     ```
-    >> apictl apply -f api-operator/apim-analytics-configs
+    >> apictl apply -f api-operator/deploy/apim-analytics-configs
```

Comment:
The config path we mentioned here is of the downloaded k8s-api-operator-x.x.x.zip
zip file in the [release](https://github.com/wso2/k8s-api-operator/releases/tag/v1.2.2).

We build this zip file by building the maven project in the root of the repository.
So the previous value `api-operator/apim-analytics-configs` should be correct

Reply:
I'll close the issue and the PR. Thanks for the update